Index: DAO/GenericDAO.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package DAO;\r\n\r\npublic interface GenericDAO<T> {\r\n    void add(T entity);\r\n    T getById(int id);\r\n    void update(T entity);\r\n    void delete(int id);\r\n\r\n}\r\n
===================================================================
diff --git a/DAO/GenericDAO.java b/DAO/GenericDAO.java
--- a/DAO/GenericDAO.java	
+++ b/DAO/GenericDAO.java	
@@ -1,6 +1,8 @@
 package DAO;
 
 public interface GenericDAO<T> {
+
+
     void add(T entity);
     T getById(int id);
     void update(T entity);
